MaterialDesignThemes.Wpf/ComboBoxPopup.cs

```diff
@@ -1,14 +1,21 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Windows;
 using System.Windows.Controls;
 using System.Windows.Controls.Primitives;
+using System.Windows.Data;
 using System.Windows.Media;
 
 namespace MaterialDesignThemes.Wpf
 {
     public class ComboBoxPopup : Popup
     {
+        public const double ContentPresenterExtend = 4;
+        public const double TopBottomMargin = 8;
```

Comment:
Yes please...if we don't have to make them part of the public API I'd rather that...in case someone becomes dependent on them somewhere down the line.